examples/17-layout-props-types/src/Layouts/Sidebar.elm

```diff
@@ -0,0 +1,97 @@
+module Layouts.Sidebar exposing (Model, Msg, Props, layout)
+
+import Effect exposing (Effect)
+import Html exposing (Html)
+import Html.Attributes exposing (class, style)
+import Html.Events
+import Layout exposing (Layout)
+import Route exposing (Route)
+import Route.Path
+import Shared
+import View exposing (View)
+
+
+type alias Props =
+    {}
+
+
+layout : Props -> Shared.Model -> Route () -> Layout () Model Msg contentMsg
+layout props shared route =
+    Layout.new
+        { init = init
+        , update = update
+        , view = view
+        , subscriptions = subscriptions
+        }
+
+
+
+-- MODEL
+
+
+type alias Model =
+    {}
+
+
+init : () -> ( Model, Effect Msg )
+init _ =
+    ( {}, Effect.none )
+
+
+
+-- UPDATE
+
+
+type Msg
+    = NoOp
```

Comment:
I would like to change all templates to be like this instead of "change me".

Reply:
I think `ExampleMsgReplaceMe` made more sense in elm-spa, when I was encouraging folks to use the simplest page for the job. Now, you might have pages that genuinely don't ever send messages, but still be using `Page.new` to prevent refactoring later.

I really like Noah Zachary Gordon's perspective on how "NoOp" in real programs can lead to frustration during debugging, because you lose information about "what happened".

Here's the original clip from their talk from "Elm in the Spring":
https://youtu.be/w6OVDBqergc?t=1221

I'll switch things to `NoOp` as suggested, but I strongly recommend using messages like `SearchInputFocused` or `ApiResponded`, even if you don't want need to update your model / send an effect.

Thanks for calling this out, @dz0ny! ❤️